src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -1,11 +1,43 @@
 import React from "react";
-import { shallow } from "enzyme";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { mount } from "enzyme";
+import { MemoryRouter } from "react-router";
+import { Route } from "react-router-dom";
+import dataDump from "testing/complete-redux-store-dump";
 
 import InfoPanel from "./InfoPanel";
 
+const mockStore = configureStore([]);
+
 describe("Info Panel", () => {
   it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<InfoPanel />);
-    expect(wrapper).toMatchSnapshot();
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/group-test"]}>
+          <Route path="/models/*">
```

Comment:
Now that we're doing this multiple places maybe we should also create a test scaffold for defining these test routes. https://github.com/canonical-web-and-design/jaas-dashboard/issues/86